.github/workflows/build.yml

```diff
@@ -62,7 +62,7 @@ jobs:
         wavpack
         ${{ matrix.vcpkg_packages_extras }}
       VCPKG_DEFAULT_TRIPLET: ${{ matrix.vcpkg_triplet }}
-      VCPKG_OVERLAY_TRIPLETS: overlay/triplets
+      #VCPKG_OVERLAY_TRIPLETS: overlay/triplets does not work 
```

Comment:
You could open an issue upstream to request this feature.

Reply:
https://github.com/microsoft/vcpkg/issues/18764